postprocess/patch-exoscale.jsonnet

```diff
@@ -0,0 +1,31 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+
+// The hiera parameters for the component
+local params = com.inventory().parameters.cert_manager;
+
+local pki_file = std.extVar('output_path') + '/pki.yaml';
+local pki_obj = com.yaml_load_all(pki_file);
+
+local deploy_file = std.extVar('output_path') + '/deployment.yaml';
+local deploy_obj = com.yaml_load(deploy_file);
+
+{
+  [if params.components.exoscale_webhook.enabled then 'pki']: [
```

Comment:
These conditions are not required if we simply disable the filter when the webhook isn't deployed.